include/zephyr/drivers/video-controls.h

```diff
@@ -33,11 +33,11 @@ extern "C" {
  * @name Control classes
  * @{
  */
-#define VIDEO_CTRL_CLASS_GENERIC	0x00000000	/**< Generic class controls */
-#define VIDEO_CTRL_CLASS_CAMERA		0x00010000	/**< Camera class controls */
-#define VIDEO_CTRL_CLASS_MPEG		0x00020000	/**< MPEG-compression controls */
-#define VIDEO_CTRL_CLASS_JPEG		0x00030000	/**< JPEG-compression controls */
-#define VIDEO_CTRL_CLASS_VENDOR		0xFFFF0000	/**< Vendor-specific class controls */
+#define VIDEO_CTRL_CLASS_GENERIC 0x00000000 /**< Generic class controls */
+#define VIDEO_CTRL_CLASS_CAMERA  0x00010000 /**< Camera class controls */
+#define VIDEO_CTRL_CLASS_MPEG    0x00020000 /**< MPEG-compression controls */
+#define VIDEO_CTRL_CLASS_JPEG    0x00030000 /**< JPEG-compression controls */
+#define VIDEO_CTRL_CLASS_VENDOR  0xFFFF0000 /**< Vendor-specific class controls */
 /**
```

Comment:
No change above?

Reply:
Can you move generic video changes to dedicated commit?

Reply:
OK, I have modified it

Reply:
@loicpoulain  OK， I have send a new PR